config.toml

```diff
@@ -45,50 +47,50 @@ time_format_blog = "Monday, January 02, 2006"
 description = "Production-Grade Container Orchestration"
 showedit = true
 
-latest = "v1.10"
+latest = "v1.11"
 
-fullversion = "v1.10.3"
+fullversion = "v1.10.5"
 version = "v1.10"
-githubbranch = "master"
-docsbranch = "master"
-deprecated = false
+githubbranch = "v1.10.5"
+docsbranch = "release-1.10"
+deprecated = true
 currentUrl = "https://kubernetes.io/docs/home/"
```

Comment:
@chenopis the Netlify error relates to this, I think. I don't understand what is wrong.

```
11:27:06 AM: ERROR 2018/06/27 18:27:06 error processing shortcode "shortcodes/deprecationwarning.html" for page "_index.html": template: shortcodes/deprecationwarning.html:7:65: executing "shortcodes/deprecationwarning.html" at <.Params.currenturl>: can't evaluate field currenturl in type interface {}
```